src/universal/modules/teamDashboard/components/ProviderRow/ProviderRow.js

```diff
@@ -73,8 +74,9 @@ const ProviderRow = (props) => {
     fontSize: ui.iconSize,
     fontWeight: 400
   };
+  const hasActivity = userCount > 0 || integrationCount > 0;
   return (
-    <Row style={{justifyContent: 'flex-start'}}>
+    <Row sansBorder={firstInList} style={{justifyContent: 'flex-start'}}>
```

Comment:
I see your point. I’ll see if I can clean this up. I think `Row` should be responsible for presenting itself with a border or not, but it is nuanced how to remember and declare that in parent contexts. I used to have to manage the context in usable component or utility classes in CSS land and then say things happened to `:first-child` or `:last-child` etc. which also got nuanced. Another option would be to have a panel separator and just declare it between rows. If we were to map over rows then we can see when the index === 1st array item and conditionally not show the separator. Still it takes consideration. I’ll take a gander!!